packages/core/storybook/stories/2_Typography.stories.mdx

```diff
@@ -14,30 +14,30 @@ The typography using inside Venice and their CSS Variables.
 | -------------------- | --------------------- |
 | `var(--font-family)` | 'Roboto', sans-serif' |
 
-| Variable                   | Font weight | Font size | Line height | Letter Spacing |
-| -------------------------- | ----------- | --------- | ----------- | -------------- |
-| `--title-giant`            | 700         | 52px      | 68px        | 25%            |
-| `--title-display`          | 700         | 46px      | 60px        | 25%            |
-| `--title-headline-xxlarge` | 700         | 42px      | 56px        | 25%            |
-| `--title-headline-xlarge`  | 700         | 36px      | 48px        | 25%            |
-| `--title-headline-large`   | 700         | 32px      | 40px        | 25%            |
-| `--title-headline-medium`  | 700         | 28px      | 36px        | 15%            |
-| `--title-headline-small`   | 700         | 26px      | 36px        | 15%            |
-| `--title-headline-xsmall`  | 700         | 22px      | 28px        | 15%            |
-| `--text-subtitle-large`    | 700         | 20px      | 28px        | 15%            |
-| `--text-subtitle-medium`   | 700         | 18px      | 24px        | 1%             |
-| `--text-body-large`        | 400         | 18px      | 28px        | 5%             |
-| `--text-body-medium`       | 400         | 16px      | 24px        | 5%             |
-| `--text-body-small`        | 400         | 14px      | 24px        | 25%            |
-| `--text-link-large`        | 700         | 18px      | 28px        | 5%             |
-| `--text-link-medium`       | 700         | 16px      | 24px        | 5%             |
-| `--text-link-small`        | 700         | 14px      | 24px        | 25%            |
-| `--button-large`           | 700         | 18px      | 28px        | 8%             |
-| `--button-medium`          | 700         | 16px      | 24px        | 8%             |
-| `--text-caption`           | 400         | 12px      | 20px        | 4%             |
-| `--text-caption-focused`   | 400         | 12px      | 20px        | 0.4%           |
-| `--text-overline`          | 400         | 11px      | 20px        | 16%            |
-| `--text-badge`             | 700         | 10px      | 16px        | 0.4%           |
+| Variable                   | Font weight | Font(Desktop) | Font(Mobile) | Letter Spacing |
```

Comment:
Suggestion: Could we describe the line in the table header? 
`Font/Line height (Desktop)`
Without it might mislead us.

Reply:
Nice, good idea